cdm_cut.sql

```diff
@@ -0,0 +1,166 @@
+/**
+ * CDM cut for DS-DETERMINED cohort
+ */
+DROP TABLE condition;
```

Comment:
`DROP TABLE if exists condition;` will make your code idempotent, and same thing for the rest of the drop statment

Reply:
https://github.com/kumc-bmi/ds-determined-tools/commit/6291b8b5f72259e80e25415085a42f87b9534399